libs/wingc/src/jsify.rs

```diff
@@ -934,21 +935,8 @@ impl<'a> JSifier<'a> {
 
 		code.add_code(self.jsify_to_inflight_type_method(&class.name, &free_vars, &referenced_preflight_types));
 		code.add_code(self.jsify_toinflight_method(&class.name, &captured_fields));
-
-		let mut bind_method = CodeMaker::default();
-		bind_method.open("_registerBind(host, ops) {");
-		for (method_name, method_refs) in refs {
-			bind_method.open(format!("if (ops.includes(\"{method_name}\")) {{"));
-			for (field, ops) in method_refs {
-				let ops_strings = ops.iter().map(|op| format!("\"{}\"", op)).join(", ");
-				bind_method.line(format!("this._registerBindObject({field}, host, [{ops_strings}]);",));
-			}
-			bind_method.close("}");
-		}
-		bind_method.line("super._registerBind(host, ops);");
-		bind_method.close("}");
-
-		code.add_code(bind_method);
+		code.add_code(self.jsify_register_bind_method(class, &refs, resource_type, false));
+		code.add_code(self.jsify_register_bind_method(class, &refs, resource_type, true));
```

Comment:
Calling a method twice with two boolean options looks a bit weird... I wonder if these should be separated into two methods? Or have the intent made clearer using an enum, e.g.

```suggestion
		code.add_code(self.jsify_register_bind_method(class, &refs, resource_type, BindMethod::Instance));
		code.add_code(self.jsify_register_bind_method(class, &refs, resource_type, BindMethod::Type));
```

(if this seems pedantic feel free to ignore)

Reply:
I agree this seems weird. I like that this is a function and not inlined. I also like that this reduces code duplication. So I added an enum with some comments for clarity.